PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD001.xml

```diff
@@ -0,0 +1,201 @@
+                                        </item>
+                                        <item xml:id="q8" n="7">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="63r" to="72v"/>                                            
+                                        </item>
+                                        <item xml:id="q9" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="73r" to="82v"/>                                            
+                                        </item>
+                                        <item xml:id="q10" n="9">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="83r" to="92v"/>                                            
+                                        </item>
+                                        <item xml:id="q11" n="10">
+                                            <dim unit="leaf">12</dim>
+                                            <locus from="93r" to="104v"/>                                            
+                                        </item>
+                                        <item xml:id="q12" n="11">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="105r" to="112v"/>                                            
+                                        </item>
+                                        <item xml:id="q13" n="12">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="113r" to="122v"/>                                            
+                                        </item>
+                                        <item xml:id="q14" n="13">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="123r" to="126v"/>                                            
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18 19">                          
+                            <ab type="pricking">Pricking is visible.</ab>
+                            <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                            <ab type="ruling">Ruling is visible.</ab>
+                            <ab type="ruling">The upper line is written above the ruling,
+                                the lower line is written above the ruling.</ab>
```

Comment:
Do you mean "above" or "below"? If it is above, I think the second part of the note can be deleted, because it is usually the case, that text is written above the ruling lines.